symbology/create-symbol-styles-from-web-styles/src/main/java/com/esri/samples/create_symbol_styles_from_web_styles/CreateSymbolStylesFromWebStylesSample.java

```diff
@@ -166,9 +167,10 @@ public void start(Stage stage) {
         });
       }
 
-      // add a map scale changed listener on the map view to control the symbol sizes at different scales
-      mapView.addMapScaleChangedListener(mapScaleChangedEvent ->
-        featureLayer.setScaleSymbols(mapView.getMapScale() >= 80000));
+      // add a map scale listener on the map view to control the symbol sizes at different scales
+      mapView.mapScaleProperty().addListener((observable, oldValue, newValue) -> {
+        featureLayer.setScaleSymbols((double) newValue >= 80000);
```

Comment:
As far as I can see this is all we can do here since feature layers don't have properties yet to bind to. 

I think @alan-edi 's point in the other PR about making artificial properties to bind to is a sensible one and we shouldn't force it. So I think the creation of String Bindings could work for the text labels in the other 2 samples building on the feedback from Alan, but can't see anything else to do here.